ml-agents/mlagents/utils/push_to_hf.py

```diff
@@ -0,0 +1,182 @@
+  - A *longer tutorial* to understand how works ML-Agents: https://huggingface.co/learn/deep-rl-course/unit5/introduction
+
+  ### Resume the training
+  ```
+  mlagents-learn <your_configuration_file_path.yaml> --run-id=<run_id> --resume
+  ```
+
+  ### Watch your Agent play
+  You can watch your agent **playing directly in your browser** 
+
+  1. If the environment is part of ML-Agents official environments, go to https://huggingface.co/unity
+  2. Step 1: Find your model_id: {repo_id}
+  3. Step 2: Select your *.nn /*.onnx file
+  4. Click on Watch the agent play 👀 
+  """
+  
+    return model_card, metadata
+
+
+def _save_model_card(repo_dir: Path, generated_model_card: str, metadata):
+    """Save a model card to the repository.
+    :param repo_dir: repository directory
+    :param generated_model_card: model card generated by _generate_model_card() method
+    :param metadata: metadata
+    """
+    readme_path = repo_dir / "README.md"
+
+    with readme_path.open("w", encoding="utf-8") as f:
+        f.write(generated_model_card)
+
+    # Save our metrics to Readme metadata
+    metadata_save(readme_path, metadata)
+
+
+def package_to_hub(run_id,
+                   path_of_run_id,
+                   repo_id: str,
+                   commit_message: str,
+                   configfile_name
+                   ):
+    """
+    This method generates the model card and upload the run_id folder
+    with all his files into the Hub
+    :param run_id : name of the run
+    :param path_of_run_id: path of the run_id folder that contains the onnx model.
+    :param repo_id: id of the model repository from the Hugging Face Hub
+    :param commit_message: commit message
+    :param configfile_name: name of the yaml config file (by default configuration.yaml)
+    """
+    print(
+        f"This function will create a model card and upload your {run_id} "
+        f"into HuggingFace Hub. This is a work in progress: If you encounter a bug, "
+        f"please send open an issue")
+
+    _, repo_name = repo_id.split('/')
+
+    # Step 1: Clone or create the repo
+    # Create the repo (or clone its content if it's nonempty)
+    api = HfApi()
+
+    repo_url = api.create_repo(
+        repo_id=repo_id,
+        exist_ok=True, )
+    
+    local_path = Path(path_of_run_id)
+
+    # Step 2: Create a config file
+    _generate_config(local_path, configfile_name)
+    
+    # Step 3: Generate and save the model card
+    generated_model_card, metadata = _generate_model_card(local_path, configfile_name, repo_id)
+    _save_model_card(local_path, generated_model_card, metadata)
+
+    print(f"Pushing repo {run_id} to the Hugging Face Hub")
+      
+    # Step 4. Push everything to the Hub
+    api.upload_folder(
+        repo_id=repo_id,
+        folder_path=local_path,
+        path_in_repo=".",
```

Comment:
```suggestion
        path_in_repo=".",
```

Not even needed if `huggingface_hub` is recent enough! (>=0.12 ?)

Reply:
Removed ✔️